scripts/bb-test-zfstests.sh

```diff
@@ -38,10 +38,12 @@ set -x
 DEFAULT_ZFSTESTS_RUNFILE=${DEFAULT_ZFSTESTS_RUNFILE:-"linux.run"}
 DEFAULT_ZFSTESTS_DISKS=${DEFAULT_ZFSTESTS_DISKS:-""}
 DEFAULT_ZFSTESTS_TAGS=${DEFAULT_ZFSTESTS_TAGS:-"functional"}
+DEFAULT_ZFSTESTS_ITERS=${DEFAULT_ZFSTESTS_ITERS:-"1"}
```

Comment:
Unlike the other `DEFAULT_` entries I don't think there's a good reason for us to allow individual builders to provide different default iterations.  Let's drop this line and move the default value in to `TEST_ZFSTESTS_ITERS` like most of the other tunings.